VotingApplication/VotingApplication.Web/Scripts/Controllers/ResultsPageController.js

```diff
@@ -1,129 +1,135 @@
 ﻿(function () {
-    angular.module('GVA.Voting').controller('ResultsPageController', ['$scope', '$location', 'PollService', function ($scope, $location, PollService) {
-        var chart;
-
-        // Turn "/#/results/abc/123" into "/#/voting/abc/123"
-        var locationTokens = $location.url().split("/");
-        locationTokens.splice(0, 2);
-        $scope.votingLink = '#/Vote/' + locationTokens.join("/");
-
-        $scope.winner = 'Lorem';
-        //Whether or not we have an "s" on the end of "Winner"
-        $scope.plural = '';
-
-        var drawChart = function (data) {
-            if (!data.length) return;
-
-            var dataUnchanged = chart && data.length === chart.series().length;
-
-            var barCount = 0;
-            for (var n = 0; n < data.length; n++) {
-                barCount++;
-                var chartSeries = chart ? chart.series() : null;
-                dataUnchanged = dataUnchanged &&
-                    chart && chart.series().length > n &&
-                    JSON.stringify(data[n].Data) == JSON.stringify(chart.series()[n].data.rawData());
+    angular.module('GVA.Voting').controller('ResultsPageController', ['$scope', '$routeParams', '$location', 'IdentityService', 'PollService', 'TokenService',
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.